contrib/ruby/test/client_test.rb

```diff
@@ -80,7 +80,7 @@ def test_trilogy_ping
     ensure_closed client
   end
 
-  def test_trilogy_ping_after_close_returns_false
+  def test_trilogy_ping_after_close_returns_raises
```

Comment:
Thanks for updating this. I think we should get rid of `returns` from the test name now

```suggestion
  def test_trilogy_ping_after_close_raises
```

Reply:
🙈 Thanks!